### ISSUE TITLE
Android

### ISSUE THREAD

**Issue by ayoubphy** [expo/expo-exposure-notification#3]
Any idea on how to work with the API for android ?

**Comment by ccheever** [comment]
Hey --

They have released the spec but I don't think its available yet to the public for usage.
Here is the spec. https://www.blog.google/documents/68/Android_Exposure_Notification_API_documentation_v1.2.pdf

**Comment by ayoubphy** [comment]
Hello,

Google has published a repo of an example app using Exposure Notification API for Android, here is the link : https://github.com/google/exposure-notifications-android

**Comment by ccheever** [comment]
thank you @ayoubphy , we'll start looking at it now

**Comment by ccheever** [comment]
we won't be able to build this or finish a module until the new version of google play services is released

**Comment by ayoubphy** [comment]
Yes I thought so too, my android devices have the beta version the google play services and it still hasn't been updated.
My knowledge in Android is limited, isn't there a way to build a perliminary version of the module and wait until google play services gets updated and released based on the implementation reference published by google ?

**Comment by ccheever** [comment]
one could stub out something based on the spec, but there would be no way to test it.
ex. looks like someone is doing that here: https://github.com/covid19risk/covidwatch-android-ag-minimal/blob/master/app/src/main/java/org/covidwatch/agminimal/FakeExposureNotification.kt

**Comment by ayoubphy** [comment]
Google released a beta of the new release of google play services, any plans on developing an android implementation ? Also, FYI : when I installed the reference sample app released by google , it throws an internal error message when I try to turn on exposure notification

**Comment by ccheever** [comment]
I believe they are only privately distributing a version of Play Services w
the relevant libraries at this time. We are trying to keep an eye on things
and find out more.

**Comment by ayoubphy** [comment]
I think they started rolling the beta for public, I have it installed on my device and it added a covid section into my google settings
![Screenshot_20200508_210929_com google android gms](https://user-images.githubusercontent.com/30992684/81449659-43517d00-9170-11ea-82fb-c6fbe5f9974e.jpg)
